tests/parser_tests/inverse.py

```diff
@@ -70,7 +70,7 @@ async def verify_inverse(parser):
                 ],
                 is_creation=False,
                 is_deletion=False,
-                rename_file_path=Path("file2.txt"),
+                rename_file_path=Path("file2.txt").resolve(),
```

Comment:
Why does this need resolve but not the other paths in the test?

Reply:
Oh, I guess you just did cwd / "file_name" for the other paths; resolve just makes it an abs path, but I'll change it to cwd / for consistency

Reply:
Right. Sorry I probably could have figured that out myself but I was reviewing from my phone. Thanks for fixing this!